pkg/bicep/testdata/simple.json

```diff
@@ -1,58 +1,223 @@
+        }
+      }
+    },
+    "enumtest": {
+      "type": "string",
+      "enum": [
+        "foo",
+        "bar"
+      ]
+    },
+    "enumtestints": {
+      "type": "integer",
+      "enum": [
+        1,
+        2
+      ]
+    },
+    "enumtestbools": {
+      "type": "boolean",
+      "enum": [
+        true,
+        false
+      ]
+    },
+    "enumtestarrays": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "enum": [
+        [
+          "foo",
+          "bar"
+        ],
+        [
+          "baz",
+          "qux"
+        ]
+      ]
+    },
+    "enumobjecttest": {
+      "type": "object",
+      "enum": [
+        {
+          "foo": "bar"
         },
-        "objecttest": {
-            "type": "object",
-            "required": ["foo"],
-            "properties": {
-                "foo": {
-                    "type": "string"
-                },
-                "bar": {
-                    "type": "integer"
-                }
-            }
+        {
+          "baz": "qux"
+        }
+      ]
+    },
+    "descriptiontest": {
+      "type": "string",
+      "description": "This is a description"
+    },
+    "descriptionenumtest": {
+      "type": "string",
+      "description": "This is a new description",
+      "enum": [
+        "foo",
+        "bar",
+        "baz"
+      ]
+    },
+    "minvaluetest": {
+      "type": "integer",
+      "minimum": 5
+    },
+    "maxvaluetest": {
+      "type": "integer",
+      "maximum": 10
+    },
+    "minmaxvaluetest": {
+      "type": "integer",
+      "minimum": 5,
+      "maximum": 10
+    },
+    "minlengthstringtest": {
+      "type": "string",
+      "minLength": 5
+    },
+    "maxlengthstringtest": {
+      "type": "string",
+      "maxLength": 10
+    },
+    "minmaxlengthstringtest": {
+      "type": "string",
+      "minLength": 5,
+      "maxLength": 10
+    },
+    "minlengtharraytest": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "minItems": 2
+    },
+    "maxlengtharraytest": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "maxItems": 5
+    },
+    "minmaxlengtharraytest": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "minItems": 2,
+      "maxItems": 5
+    },
+    "defaultstringtest": {
+      "type": "string",
+      "default": "foo"
+    },
+    "defaultintegertest": {
+      "type": "integer",
+      "default": 5
+    },
+    "defaultbooltest": {
+      "type": "boolean",
+      "default": true
+    },
+    "defaultarraytest": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "default": [
+        "foo",
+        "bar"
+      ]
+    },
+    "defaultobjecttest": {
+      "type": "object",
+      "required": [
+        "bar",
+        "foo"
+      ],
+      "properties": {
+        "bar": {
+          "type": "string"
         },
-        "nestedtest": {
```

Comment:
You removed this test, but I don't see anything that tests a nested object.

Reply:
It's not removed, it's just sitting on L45 now.